app/assets/stylesheets/application.css.scss

```diff
@@ -125,16 +141,51 @@ body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
   height: 100vh;
   background: $dark-bg;
   transition: all .5s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+
   .menu-burger {
     width: 40px;
     height: 40px;
-    border: 2px solid #f9f9f9;
+    border: 2px solid $muteWhite;
     right: 0;
     transition: 0.2s;
+
+    .icon-bar{
+      transition: 0.4s;
+    }
+
+    #top{
+      margin-top: 7px;
+      border-bottom: none;
+      border-top: 10px solid $bun;
+      border-radius: 30px 30px 0 0;
+      height: 9px;
+      width: 22px;
+      transition: 0.2s;
+    }
+
+    #meat{
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space